commit c93da7d9cf161ffda2366a96eb060c3b824cb549
Author: Roman Lebedev <lebedev.ri@gmail.com>
Date:   Tue Sep 28 18:23:17 2021 +0300

    Revert "[CMake] Enable LLVM_ENABLE_PER_TARGET_RUNTIME_DIR by default on Linux"
    
    See original review https://reviews.llvm.org/D107799
    
    This reverts commit f9dbca68d48e705f6d45df8f58d6b2ee88bce76c.

Index: llvm-toolchain-snapshot_14~++20211002104316+f33274c7bf44/llvm/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_14~++20211002104316+f33274c7bf44.orig/llvm/CMakeLists.txt
+++ llvm-toolchain-snapshot_14~++20211002104316+f33274c7bf44/llvm/CMakeLists.txt
@@ -677,7 +677,13 @@ endif()
 set(LLVM_TARGET_TRIPLE_ENV CACHE STRING "The name of environment variable to override default target. Disabled by blank.")
 mark_as_advanced(LLVM_TARGET_TRIPLE_ENV)
 
-set(LLVM_ENABLE_PER_TARGET_RUNTIME_DIR OFF CACHE BOOL
+# Per target dir not yet supported on Arm 32 bit due to arm vs armhf handling
+if(CMAKE_SYSTEM_NAME MATCHES "Linux" AND NOT CMAKE_SYSTEM_PROCESSOR MATCHES "^arm")
+  set(LLVM_ENABLE_PER_TARGET_RUNTIME_DIR_default ON)
+else()
+  set(LLVM_ENABLE_PER_TARGET_RUNTIME_DIR_default OFF)
+endif()
+set(LLVM_ENABLE_PER_TARGET_RUNTIME_DIR ${LLVM_ENABLE_PER_TARGET_RUNTIME_DIR_default} CACHE BOOL
   "Enable per-target runtimes directory")
 
 set(LLVM_PROFDATA_FILE "" CACHE FILEPATH
